tools/JsonCli/src/Microsoft.TemplateEngine.JsonCli.csproj

```diff
@@ -1,13 +1,13 @@
 <Project Sdk="Microsoft.NET.Sdk">
 
-  <PropertyGroup>
-    <OutputType>Exe</OutputType>
-    <TargetFramework>netcoreapp2.0</TargetFramework>
-  </PropertyGroup>
+    <PropertyGroup>
+        <OutputType>Exe</OutputType>
+        <TargetFrameworks>netcoreapp2.0;netcoreapp3.0</TargetFrameworks>
```

Comment:
Just to be sure, but I'm assuming these changes are coming from something else that you're copying in, right?

Reply:
Most of the changes, yes